user_docs/en/changes.t2t

```diff
@@ -26,6 +26,7 @@ This option now announces additional relevant information about an object when t
 - The SAPI4 synthesizer now properly supports volume, rate and pitch changes embedded in speech. (#15271)
 - Multi line state is now correctly reported in applications using Java Access Bridge. (#14609)
 - In LibreOffice, words deleted using the ``control+backspace`` keyboard shortcut are now also properly announced when the deleted word is followed by whitespace (like spaces and tabs). (#15436)
+- In Microsoft Excel braille is updated and active cell with its content is spoken with UIA disabled when ``control+y``, ``control+z`` or ``alt+backspace`` is pressed. (15547)

```

Comment:
Since UIA is disabled by default for Excel, is this still accurate? I had trouble parsing your message

```suggestion
- In Microsoft Excel, braille is updated, and the active cell content is spoken, when ``control+y``, ``control+z`` or ``alt+backspace`` is pressed. (15547)
```

Reply:
It does not work if UIA is enabled (at least it does not work in windows 10 22h2 and excel 2019). If you think entry should be changed let me know.

Reply:
I've slightly updated my suggestion to explicitly mention UIA. Can you confirm it's still accurate?